### PR TITLE
Added Docker scripts in the new Docker directory

### PR DESCRIPTION
**Pull request by jhsu802701**
#### What? Why?
These changes make it so much easier to use Docker.  You no longer have to keep typing "docker-compose run --rm web".

#### What should we test?
This pull request adds scripts in the "docker" directory and updates the documentation.  The mechanisms behind the app are unchanged.

#### Release notes
Added scripts to make it easier to use Docker.  You no longer have to keep typing "docker-compose run --rm web".

#### Documentation updates
The URL leading to the Docker page should be updated.

### CODE REVIEW

**Review by luisramos0 — COMMENTED** [review]
This looks very good, thank you!
I want to give this a try before I approve.

I think that at the end of the DOCKER.md we could have a bullet list of the scripts available and what each of them does in very few words.

**Comment by jhsu802701** [comment]
I added a summary of the Docker scripts to the end of the docker/README.md file (which replaces DOCKER.md) .  This pull request already includes an updated link to the Docker page in GETTING_STARTED.md.

File: docker/README.md
Comment (luisramos0) [inline docker/README.md:54]: nice!
can we order this in the order of typical usage?
- build
- server
- test
- exec
- seed
- nuke

**Review by luisramos0 — APPROVED** [review]
Nice :ok_hand:

I think I'd remove testc and testm, there are too slow. devs will always run specific specs when they are preparing PRs.

**Comment by jhsu802701** [comment]
I removed docker/testc and docker/testm and made your requested changes to the order of the scripts listed in the summary section.

**Review by luisramos0 — APPROVED** [review]
:ok_hand:

**Comment by jhsu802701** [comment]
Is there anything else I should add, remove, or change?

**Comment by sigmundpetersen** [comment]
Hey @jhsu802701 , this just needs an approval from another reviewer and will then move on to QA 👍

**Review by sauloperez — DISMISSED** [review]
The only bit from the decorator I couldn't find is https://github.com/openfoodfoundation/openfoodnetwork/blob/427d1aaeff51234b8995351f555f1cfd7bb9bb2d/app/models/spree/order_decorator.rb#L32-L40. Where did that go?

File: docker/cop
Comment (sauloperez) [inline docker/cop:8]: we're missing `--rm` after `run`, right?

File: docker/nukec
Comment (sauloperez) [inline docker/nukec:3]: it'll destroy all running containers, right? I'd clarify that in that case.

File: docker/nukec
Comment (sauloperez) [inline docker/nukec:7]: but that will destroy all networks, including any other not related to OFN?

File: docker/seed
Comment (sauloperez) [inline docker/seed:24]: aren't we missing `--rm` here too? why don't reuse the script you added?

File: docker/exec
Comment (sauloperez) [inline docker/exec:1]: what concerns me a bit of this command is that it mixes the meanings of docker-compose's exec and run. Do you see any problem renaming it to `docker/run` so we stick to that meaning?

File: docker/cop
Comment (sauloperez) [reply docker/cop:8]: btw, -D seems to be the default already

```
-D, --[no-]display-cop-names     Display cop names in offense messages.
                                                   Default is true.
```

**Comment by jhsu802701** [comment]
- I renamed docker/exec to docker/run and updated the docker/README.md file to reflect this.

- I moved the "docker prune network -f" command from docker/nukec to docker/nuke.

- I corrected the "docker-compose" commands in the scripts in the Docker directory.

- I updated the docker/cop script to remove the "-D" flag (due to the redundancy).

- The contents of the DOCKER.md file (including the section on troubleshooting Windows) have been moved to the docker/README.md file.

**Comment by jhsu802701** [comment]
Are there any additional changes I need to make?  Until this pull request is merged, people will have to keep typing "docker-compose run --rm web".

**Review by sauloperez — APPROVED** [review]
I just took a look and it works. Thanks for this contribution @jhsu802701 !

**Pull request merged**
